### ISSUE TITLE
Loss increase dramatically after 60000 iterations

### ISSUE THREAD

**Issue by qiuqiangkong** [deepsound-project/samplernn-pytorch#33]
Hi, really brilliant code! when I run the pytorch implementation, I found the trianing and validation loss increases dramtically after 60000 iterations. The loss curve looks like: https://drive.google.com/open?id=1_64-jD3hOtXmrOoVMq5hD8pWfmEUc7yE

Do you have any idea of this increased loss? Thank you very much! 

Qiuqiang
